internal/resources/tanzupackagerepository/data_source_repository_test.go

```diff
@@ -68,7 +68,7 @@ func TestAcceptanceForPackageRepositoryDataSource(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				PreConfig: func() {
-					if testConfig.ScopeHelperResources.Cluster.KubeConfigPath == "" && found {
```

Comment:
What was the found variable check done here?

Reply:
found is for this env variable `ENABLE_PKGREPO_ENV_TEST` if it's not found let say in case of mock acceptance then this check preventing test from skipping, cause we don't required to pass kubeconfig  in mock acceptance test.